tests/helpers/test_script.py

```diff
@@ -3012,7 +3012,7 @@ async def test_parallel(hass: HomeAssistant, caplog: pytest.LogCaptureFixture) -
 
     expected_trace = {
         "0": [{"result": {}}],
-        "0/parallel/0/sequence/0": [
+        "0/parallel/0/0": [
```

Comment:
This doesn't feel right. I agree that `sequence` is the wrong word, but it needs to have some name to make these paths be more readable. What about `parallel` ?

Reply:
hm no, that is the parent name 🤔

Reply:
It's just another script/action (which we generally refer to as sequence). 
IMHO `sequence` is the exact right work here.

You can run multiple sequences of actions in parallel.

Reply:
Quick example, before this PR, this:

```yaml
parallel:
  - event: "event child 1"
  - sequence:
    # These events below are sequential, not parallel.
    - event: "event child 2, step 1"
    - event: "event child 2, step 2"
  - event: "event child 3"
```

Results in:

```
0/parallel/0/sequence/0  # event child 1
0/parallel/1/sequence/0  # event child 2, step 1
0/parallel/1/sequence/1  # event child 2, step 2
0/parallel/2/sequence/0  # event child 3
```

Reply:
Then I suggest we keep it sequence.

Reply:
In that case, we now have a frontend bug 👯